packages/client/src/client.js

```diff
@@ -504,6 +504,24 @@ export class PercyClient {
     await this.finalizeComparison(comparison.data.id);
     return comparison;
   }
+
+  tokenType() {
+    let token = this.getToken();
+
+    const type = token.split('_')[0];
+    switch (type) {
+      case 'auto':
+        return 'automate';
+      case 'web':
+        return 'web';
+      case 'app':
+        return 'app';
+      case 'ss':
+        return 'generic';
+      default:
+        return 'web';
+    }
+  }
```

Comment:
Don't add this here, rebase your branch on top of https://github.com/percy/cli/pull/1270/